src/App.tsx

```diff
@@ -1,7 +1,41 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  myKey: string | null,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    myKey: null,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.lastKeyClick);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.lastKeyClick);
+  }
+
+  lastKeyClick = (elem: KeyboardEvent) => {
+    this.setState({
+      myKey: elem.key,
+    });
+  };
+
+  render(): React.ReactNode {
+    return (
+      <div className="App">
+        {this.state.myKey
+          ? (
+            <p className="App__message">
+              The last pressed key is [
+              {this.state.myKey}
+              ]
+            </p>
+          )
+          : (<p className="App__message">Nothing was pressed yet</p>)}
```

Comment:
If you write an entire element in a single string you can avoid wrapping it with parentheses:
```
: <p className="...">...</p>
```

Reply:
Okey